yoti_python_sdk/version.py

```diff
@@ -0,0 +1,2 @@
+# -*- coding: utf-8 -*-
+__version__ = "2.6.0"
```

Comment:
What's the rationale behind putting this in its own file?  Is it somehow part of the build?

Reply:
Yes it is part of the build. If we are wanting to define the version in one place, we need a way of accessing it during `__init__.py`. So during `__init__.py`, we read it from the file, since we need to know the version **before** we initalise it (and then could use `yoti_python_sdk.__version__`. 

I didn't think it would be efficient or necessary to put anything else in this file.